<launch>

    <node pkg="prediction_kf" type="prediction_kf_node" name="prediction_kf_node" output="screen">
        <param name="visual_topic" type="string" value="/pnp_twist"/>
        <param name="publisher_topic" type="string" value="/prediction_kf/predict"/>
        <param name="debug_topic" type="string" value="/prediction_kf/debug"/>
        <param name="position_weight" type="double" value="2000.0"/>
        <param name="velocity_weight" type="double" value="10000.0"/>
        <param name="chi_square_threshold" type="double" value="1000.0"/>
        <param name="repropagate_time" type="int" value="5"/>
    </node>
    <node pkg="rosbag" type="play" name="static_target_moving_chassis"
        output="screen" args="-s 0 /home/ros/Downloads/2018-06-21-12-11-35.bag -q"/>
</launch>
